source/utilities.cc

```diff
@@ -146,7 +166,12 @@ namespace aspect
 
                   // Assign all the elements to the "all" value
                   for (unsigned int field_index=0; field_index<n_fields; ++field_index)
-                    return_values[field_index] = Utilities::string_to_double(key_and_value[1]);
+                    {
+                      return_map[field_index].push_back(Utilities::string_to_double(key_and_value[1]));
+
+                      if (subentry_structure)
+                        (*subentry_structure)[field_index] = 1;
+                    }
```

Comment:
So I guess that mean we don't allow entries in the format `all:100|200`. Is that something we want, or, alternatively, should we state somewhere you can't do that?